models/battleship/ship.go

```diff
@@ -40,3 +42,7 @@ func (sh *Ship) GotHit() {
 func (sh *Ship) IsSunk() bool {
 	return sh.hits == sh.length
 }
+
+func (sh *Ship) GetHitCoords() []Coords {
+	return sh.coordsHits
```

Comment:
nit: `hitCoordinates` is better naming

Reply:
Ahhh you're right, thanks for being nit picky, I want it!